app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="user">
+            <img src="<%= user.photo %>" class="image">
+            <div class="user-container">
+            <h1 class="user-name"><%= link_to user.name, user_path(user) %></h1>
+            <p class="number-posts">Number of posts: <%= user.posts.count %></p>
+            </div>
+            </li>
```

Comment:
- [ ] [REQUIRED] Excellent job done @sanieni6 🥇 , however, we need to ensure that this partial matches the given wireframe. Let's separate the div for the user image from the `div` that houses the User's name and the number of posts then you can flex the two `div`s. Also, do not forget to position the Number of posts as it is positioned in the given wireframe.
### Your design
<img width="1106" alt="Screenshot 2023-06-14 at 23 48 55" src="https://github.com/sanieni6/BlogApp/assets/58692590/78729e97-b112-4b87-83f5-8d69789b3e62">

### Wireframe
<img width="526" alt="Screenshot 2023-06-14 at 23 53 35" src="https://github.com/sanieni6/BlogApp/assets/58692590/078f71a8-5669-4005-86d8-aa51628c25eb">

Reply:
well resolved! ✔️